app/models.py

```diff
@@ -198,3 +199,8 @@ async def get_provider_for_key(db: Database, api_key_hash: str) -> Union[str, No
         query = query.where(providers.c.api_key_hash == api_key_hash)
         provider = await db.fetch_one(query)
         return provider[0] if provider else None
+
+class TagsEnum(str, Enum):
```

Comment:
I just noticed this class would be a better fit for the "schemas" file, since models is reserved for actual interactions with the DB.